src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.github.lindenb.jbwa.jni.AlnRgn;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.CigarUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.List;
+import java.util.Objects;
+
+class AlignmentRegion {
+
+    final String contigId;
+    final String breakpointId;
```

Comment:
To me, because the design now is such that `AssembledBreakpoint` is composed of `AlignmentRegion`'s, but `AssembledBreakpoint` doesn't have such a field as `breakpointId` whereas `AlignmentRegion`'s have, it makes sense to move the `breakpointId` field to `AssembledBreakpoint` class.